static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -28,42 +23,43 @@ export enum RuleAction {
   CREATE_ALERT_LATER = 2,
 }
 
-const ISSUE_ALERT_DEFAULT_ACTION: Omit<
-  IssueAlertRuleAction,
-  'label' | 'name' | 'prompt'
-> = {
-  id: IssueAlertActionType.NOTIFY_EMAIL,
-  targetType: 'IssueOwners',
-  fallthroughType: 'ActiveMembers',
-};
-
-const METRIC_CONDITION_MAP = {
-  [MetricValues.ERRORS]: IssueAlertConditionType.EVENT_FREQUENCY,
-  [MetricValues.USERS]: IssueAlertConditionType.EVENT_UNIQUE_USER_FREQUENCY,
-} as const;
+function isRuleAction(val: number): val is RuleAction {
+  return Object.values(RuleAction).includes(val);
+}
 
-type StateUpdater = (updatedData: RequestDataFragment) => void;
-type Props = DeprecatedAsyncComponent['props'] & {
-  onChange: StateUpdater;
-  organization: Organization;
-  alertSetting?: string;
-  interval?: string;
-  metric?: MetricValues;
-  notificationProps?: IssueAlertNotificationProps;
-  platformLanguage?: SupportedLanguages;
-  threshold?: string;
-};
+function parseRuleAction(val: number | string) {
+  const ruleAction = parseInt(String(val), 10);
+  if (isRuleAction(ruleAction)) {
+    return ruleAction;
+  }
+  throw new RangeError('Supplied alert creation action is not handled');
+}
 
-type State = DeprecatedAsyncComponent['state'] & {
-  alertSetting: string;
-  // TODO(ts): When we have alert conditional types, convert this
-  conditions: any;
-  interval: string;
-  intervalChoices: Array<[string, string]> | undefined;
-  metric: MetricValues;
+function metricValueToConditionType(metricValue: MetricValues): IssueAlertConditionType {
+  switch (metricValue) {
+    case MetricValues.ERRORS:
+      return IssueAlertConditionType.EVENT_FREQUENCY;
+    case MetricValues.USERS:
+      return IssueAlertConditionType.EVENT_UNIQUE_USER_FREQUENCY;
+    default:
+      throw new RangeError(`Supplied metric value ${metricValue} is not handled`);
+  }
+}
 
-  threshold: string;
-};
+const METRIC_CHOICES = [
+  {value: MetricValues.ERRORS, label: t('occurrences of')},
+  {value: MetricValues.USERS, label: t('users affected by')},
+];
+
+const INTERVAL_CHOICES = [
+  {value: '1m', label: t('one minute')},
+  {value: '5m', label: t('5 minutes')},
+  {value: '15m', label: t('15 minutes')},
+  {value: '1h', label: t('one hour')},
+  {value: '1d', label: t('one day')},
+  {value: '1w', label: t('one week')},
+  {value: '30d', label: t('30 days')},
+];
```

Comment:
The output of fetching `/rule-conditions/` and all the utils like `unpackConditions` & `getAvailableMetricOptions` & `onLoadAllEndpointsSuccess` was to extract these, and only these, constants that've been around for ~10years.